MINT.postman_collection.json

```diff
@@ -103,7 +103,7 @@
 						"body": {
 							"mode": "graphql",
 							"graphql": {
-								"query": "mutation updateModelPlan ($id: UUID!, $changes: ModelPlanChanges!) {\n    updateModelPlan (id: $id, changes: $changes) {\n        id\n        modelName\n        abbreviation\n        archived\n        createdBy\n        createdDts\n        modifiedBy\n        modifiedDts\n        status\n        suggestedPhase {\n            phase\n            suggestedStatuses\n        }\n        isFavorite\n        isCollaborator\n        opSolutionLastModifiedDts\n    }\n}",
+								"query": "mutation updateModelPlan ($id: UUID!, $changes: ModelPlanChanges!) {\n    updateModelPlan (id: $id, changes: $changes) {\n        id\n        modelName\n        abbreviation\n        archived\n        createdBy\n        createdDts\n        modifiedBy\n        modifiedDts\n        status\n        taskListStatus\n        suggestedPhase {\n            phase\n            suggestedStatuses\n        }\n        previousSuggestedPhase\n        isFavorite\n        isCollaborator\n        opSolutionLastModifiedDts\n    }\n}",
```

Comment:
Note that the proper current variable name is previousSuggestedPhase, not anticipatedStatus. This is a carryover from 1905. If a three way merge is not automatic, we can simply take this version.